### ISSUE TITLE
File structure in tests

### ISSUE THREAD

**Issue by ElectricMaxxx** [sonata-project/SonataDoctrinePhpcrAdminBundle#457]
I would suggest to create  new structure for the tests and to create Webtests. So lets do something like:

```
Tests/
   Unit/
        ... tests we currently do.
   WebTests/
```

**Comment by dbu** [comment]
good idea. 

additionally i would love to move php code out of the repo root and use src/ and tests/ folders. but i guess we conflict with dev-kit for that... wdyt @Soullivaneuh , can we do that? or is that something you don't want in sonata repositories?

**Comment by ElectricMaxxx** [comment]
I think the file structure should be done before releasing some stuff. Adding WebTests would be simply a nice to have, right?

@greg0ire what about moving code into `src/` folder as done in symfony and symfony cmf?

**Comment by greg0ire** [comment]
I'm all for both changes, even on bundles. Can you open a dedicated issue on dev-kit?

**Comment by ElectricMaxxx** [comment]
I created #467 as an follow up, cause the creation is a nice to have on version 2.0 only. The file structure change is a must have befor releasing.

**Comment by jordisala1991** [comment]
You already have this structure: https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/tree/2.x/tests

Related to #468 

Closing
